.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
Consider whether you really want to exclude the entire `media/` directory. If this directory contains media files that are part of your application's assets, you might want to include it in version control. If it contains user-uploaded files or other content that should not be tracked, then excluding it is appropriate.